clients/src/main/java/org/apache/kafka/clients/consumer/internals/Fetcher.java

```diff
@@ -288,6 +291,20 @@ private void throwIfOffsetOutOfRange() throws OffsetOutOfRangeException {
     }
 
     /**
+     * If any topic from previous fetchResponse contatains Authorization error, throw ApiException.
+     * @throws ApiException
+     */
+    private void throwIfUnauthorized() throws ApiException {
+        if (!unauthorizedTopicPartition.isEmpty()) {
+            StringBuilder sb = new StringBuilder();
+            for (TopicPartition topicPartition: unauthorizedTopicPartition)
```

Comment:
Fixed. I thought in java our checkstyle would report things like this, guess we need to update our template.